coordinator/gscoordinator/cluster_builder.py

```diff
@@ -400,7 +429,23 @@ def get_engine_headless_service(self):
     def get_learning_service(self, object_id, start_port):
```

Comment:
This function is used with a new name "get_graphlearn_service" in kubernetes_launcher.py line 878

Reply:
fixed